sql/core/benchmarks/SubExprEliminationBenchmark-results.txt

```diff
@@ -0,0 +1,15 @@
+================================================================================================
+Benchmark for performance of subexpression elimination
+================================================================================================
+
+Preparing data for benchmarking ...
+OpenJDK 64-Bit Server VM 1.8.0_265-b01 on Mac OS X 10.15.6
+Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
+from_json as subExpr:                      Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-------------------------------------------------------------------------------------------------------------------------
+subexpressionElimination on, codegen on             2303           2543         238          0.0    23029833.1       1.0X
+subexpressionElimination on, codegen off           23107          23520         427          0.0   231069443.3       0.1X
+subexpressionElimination off, codegen on           23363          23848         421          0.0   233634044.9       0.1X
+subexpressionElimination off, codegen off          22997          23355         438          0.0   229974135.0       0.1X
```

Comment:
If we are going to merge this first, we need to merge a subset of this PR. Only the last two.
```
subexpressionElimination off, codegen on           23363          23848         421          0.0   233634044.9       0.1X
subexpressionElimination off, codegen off          22997          23355         438          0.0   229974135.0       0.1X
```

Reply:
Why? We don't need `subexpressionElimination on` cases?